test/living-html/files/current-script.js

```diff
@@ -0,0 +1,6 @@
+// jshint ignore: start
```

Comment:
Looks like `jshint browser: true` does the trick. I'm sort of confused why this isn't happening on other (similar) files though.